plugins/inputs/rabbitmq/README.md

```diff
@@ -115,6 +116,12 @@ For additional details reference the [RabbitMQ Management HTTP Stats](https://cd
 
 ### Sample Queries:
 
+Message rates for the entire node can be calculated from total message counts. For instance, to get the rate of messages published per minute, use this query:
+
+```
+SELECT NON_NEGATIVE_DIFFERENCE(LAST("messages_published")) AS messages_published_rate
+FROM rabbitmq_overview WHERE time > now() - 10m GROUP BY time(1m)
```

Comment:
I might recommend `non_negative_derivative(..., 1m)` instead of `difference()`. That way the query always gives you the 1m rate, regardless of the group by interval. Both methods are perfectly valid, just the derivative tends to be more useful in graphing as your group by interval changes based on the time range.

Reply:
Alright.